Code/DataStructs/BitOps.cpp

```diff
@@ -193,7 +197,41 @@ int NumOnBitsInCommon(const T1& bv1, const T2& bv2) {
   return static_cast<int>(OnBitsInCommon(bv1, bv2).size());
 }
 
+namespace {
+struct bitset_impl {
+  std::vector<unsigned long> m_bits;
+  std::size_t m_num_bits;
+};
+
+bool canUseBitmapHack(){
```

Comment:
size of is a constant expression, I don't think it needs to be in a function.

i.e.
```
namespace {
const bool CanUseBitmapHack = sizeof(boost::dynamic_bitset<>) == sizeof(bitset_impl);
}
```